src/database/testcases.tsx

```diff
@@ -0,0 +1,196 @@
+import { deleteFacility, saveFacility } from "./index";
+
+// 1. Full Form Submission - New Facility
+//
+// Note: some fileds get submitted along with the form submission
+// that aren't actually valid model inputs (i.e. localeDataSource)
+saveFacility({
+  name: "Ascension Correctional Facility",
+  modelInputs: {
+    stateCode: "Louisiana",
+    countyName: "Ascension",
+    rateOfSpreadFactor: "moderate",
+    usePopulationSubsets: true,
+    facilityOccupancyPct: 0.85,
+    facilityDormitoryPct: 0.25,
+    hospitalBeds: 24,
+    localeDataSource: {},
+    totalIncarcerated: 480,
+    ageUnknownPopulation: 305,
+    ageUnknownCases: 3,
+    confirmedCases: 451,
+    staffCases: 3,
+    staffPopulation: 37,
+    plannedReleases: [
+      {
+        date: "2020-04-24T04:00:00.000Z",
+        count: 100,
+      },
+      {
+        date: "2020-05-01T04:00:00.000Z",
+        count: 50,
+      },
+    ],
+    age0Cases: 2,
+    age20Cases: 1,
+    age45Cases: 3,
+    age55Cases: 1,
+    age65Cases: 1,
+    age75Cases: 1,
+    age85Cases: 1,
+    age85Population: 25,
+    age75Population: 25,
+    age65Population: 25,
+    age55Population: 25,
+    age45Population: 25,
+    age20Population: 25,
+    age0Population: 25,
+  },
+});
+
+// Verified to work as expected,  ✅
```

Comment:
And `localeDataSource` was not persisted as desired

Reply:
Maybe add this comment explicitly in your code here, if we follow my suggestion to leave this in the repository as test code.